kw

```diff
@@ -50,17 +62,27 @@ if [ -f "${KW_BASE_DIR}/src/lib/kwlib.sh" ]; then
   KW_ETC_DIR="${KW_BASE_DIR}/etc"
   KW_PLUGINS_DIR="${KW_LIB_DIR}/plugins"
   # KW_CACHE_DIR # use default cache folder
-fi
-##END-DEV-MODE##
 
-# Export external variables required by kworkflow
-export KWORKFLOW
+  # include IO utilities
+  include "${KW_LIB_DIR}/lib/kwio.sh"
 
-#INJECT_CODE_TRACING_SETUP
+  # Print a notice to the developers so he knows some envvars have been changed
+  # We redirect to stderror because this output would make tests fails if the developer is testing
+  say "[INFO]: Running kw in developer mode. Envvars set to the following:
 
-# This is the one and only time a file will be sourced this way.
-# The include function (sourced from this file) should always be used for file sourcing.
-. "${KW_LIB_DIR}/lib/kw_include.sh" --source-only
+KW_CACHE_DIR =${KW_CACHE_DIR}
+KW_DATA_DIR=${KW_DATA_DIR}
+KW_DB_DIR=${KW_DB_DIR}
+KW_DOC_DIR=${KW_DOC_DIR}
+KW_ETC_DIR=${KW_ETC_DIR}
+KW_LIB_DIR=${KW_LIB_DIR}
+KW_MAN_DIR=${KW_MAN_DIR}
+KW_PLUGINS_DIR=${KW_PLUGINS_DIR}
+KW_SOUND_DIR=${KW_SOUND_DIR}
```

Comment:
See if you can break the lines conform the [code style](https://kworkflow.org/content/codingstyle.html#breaking-long-lines-and-strings). To add newlines use something like the following:

```bash
dev_mode_message='[INFO]: Running kw in developer mode.'$'\n'
dev_mode_message+='kw environment variables set to the following:'$'\n'
dev_mode_message+="KW_CACHE_DIR =${KW_CACHE_DIR}"$'\n'
...

printf '%s\n' "$dev_mode_message"
```